source/docs/one-command-deployment.md

```diff
@@ -314,6 +314,46 @@ After a few moments, your website will be deployed.
 
 For deployment-related considerations, you can refer to [Our documentation](https://github.com/NaturalSelectionLabs/hexo-deployer-rss3/blob/develop/README.md).
 
+## Layer0
+
+[Layer0](https://www.layer0.co) is an all-in-one platform to develop, deploy, preview, experiment on, monitor, and run your headless frontend, focused on EdgeJS, predictive prefetching, and performance monitoring.
+
+1. In your hexo project directory, run:
+```bash
+npm i -g @layer0/cli
+0 init
+```
+
+2. Update routes.js (created by 0 init):
+```js
+// This file was added by layer0 init.
+// You should commit this file to source control.
+
+import { Router } from '@layer0/core/router'
+
+export default new Router().static('public', ({ cache }) => {
+  cache({
+    edge: {
+      maxAgeSeconds: 60 * 60 * 60 * 365,
+      forcePrivateCaching: true,
+    },
+    browser: {
+      maxAgeSeconds: 0,
+      serviceWorkerSeconds: 60 * 60 * 24,
+    },
+  })
+})
+```
+
+3. Deploy
+```bash
+npm run build
```

Comment:
Hexo doesn't use npm run build directly. So what does `npm run build` here for?

Reply:
Thanks for pointing that out @SukkaW, was referencing to the [demo](https://github.com/layer0-docs/layer0-hexo-example/blob/main/package.json) while I forgot to include the example. Updated to run `hexo generate` instead.